### ISSUE TITLE
If a Document fails to process; ingesting should continue

### ISSUE THREAD

**Issue by i00** [imartinez/privateGPT#442]
Note: if you'd like to *ask a question* or *open a discussion*, head over to the [Discussions](https://github.com/imartinez/privateGPT/discussions) section and post it there.

**Describe the bug and how to reproduce it**
If a document fails to load it crashes the ingest process

**Expected behavior**
The process should continue on and list the documents at the end

**Environment (please complete the following information):**
 - OS / hardware: Windows 11
 - Python version 3.11
- Other relevant information

**Additional context**

**Comment by schwab** [comment]
I agree with this issue. In my case I'm loading a bunch of .md files into the source_documents and when only one of them fails, they all stop processing. To attempt to handle this, I added a try/except in the MyElmLoader class, but this reveals that all of them actually fail to load with an exception like : amp-judges-21.md: File is not a zip file.  This is ODD though because the text files are very simple and I no idea why it expects them to be zip files.  

I have traced it as far down as the UnstructuredMarkdownLoader which is doing an import like : 
``` python
from unstructured.partition.md import partition_md
```
This is in the _get_elements method. That import raises the exception which causes things to die, but why it wants a zip file I do not know yet.  Maybe it expects some kind of configuration to be there, but IDK.
